azerothcore/azerothcore-wotlk
# fix(Core/Spells): Fixed implementation of CAST_FLAG_POWER_LEFT_SELF f…

…lag.

Do not send power update packets when gaining power via energize spell effects to match retail behavior (source: TrinityCore).
Fixes #10820



## Issues Addressed:

- Closes #10820
- Closes https://github.com/chromiecraft/chromiecraft/issues/3100

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

Use Judgement of Light/Wisdom without any MP5.
Check mana regen on judgment proc.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
